src/main/resources/META-INF/plugin.xml

```diff
@@ -35,15 +35,16 @@
     ]]>
   </change-notes>
 
-
-
   <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
   <idea-version since-build="181.3741.2"/>
```

Comment:
Should we align this version with the one used by gradle to build the extension ?

Reply:
Yes

Reply:
done. using 182.3684.101 version